pystac_monty/sources/glide.py

```diff
@@ -165,7 +165,7 @@ def get_hazard_detail(self, item: Item) -> HazardDetail:
 
         return HazardDetail(
             cluster=self.hazard_profiles.get_cluster_code(item),
-            severity_value=magnitude or 0,
+            severity_value=float(magnitude) if magnitude.isnumeric() else 0,
```

Comment:
Add a note regarding it. If the alphanumeric value is present in the magnitude, it is converted to 0 for now. But the actual value of the magnitude can be found in the properties.magnitude.